digital_comms/ccam.py

```diff
@@ -128,7 +221,16 @@ def capacity(self):
         return summed_capacity / len(self._pcd_sectors)
 
     def demand(self):
-        """Return the mean capacity demand from all nested postcode sectors
+        """Calculate demand as the mean capacity demand from all nested postcode sectors
```

Comment:
Should be: """Calculate demand per square kilometer (Mbps km^2) from all nested postcode sectors."""

Reply:
Done